rewards/tx_utils.py

```diff
@@ -41,9 +42,9 @@ def get_gas_price_of_tx(
         EMISSIONS_CONTRACTS[chain]["GasOracle"], abi_name="ChainlinkOracle", chain=chain
     )
 
-    if chain == "eth":
+    if chain == Network.Ethereum:
         gas_price_base = Decimal(tx_receipt.get("effectiveGasPrice", 0) / 10 ** 18)
-    elif chain in ["polygon", "arbitrum"]:
+    elif chain in [Network.Polygon, Network.Arbitrum]:
         tx = web3.eth.get_transaction(tx_hash)
         gas_price_base = Decimal(tx.get("gasPrice", 0) / 10 ** 18)
```

Comment:
we should likely set a constant with the gas_decimals and just use that instead of hardcoding 10 ** 18 here